pkg/flowaggregator/flowaggregator.go

```diff
@@ -724,3 +815,110 @@ func (fa *flowAggregator) createInfoElementForTemplateSet(ieName string, enterpr
 	}
 	return ie, nil
 }
+
+func (fa *flowAggregator) watchConfiguration() {
+	klog.Info("Watching for FlowAggregator configuration file")
+	for {
+		select {
+		case event, ok := <-fa.configWatcher.Events:
+			if !ok {
+				return
+			}
+			// When mounted configmap is updating, watcher event "CHMOD" and "REMOVE" will happen.
+			// Need to add configFile back to watcher in order to watch the updated configMap.
+			if event.Op == fsnotify.Remove {
+				if err := fa.configWatcher.Add(fa.configFile); err != nil {
+					klog.Fatal(err)
+				}
+			}
```

Comment:
do you think we should filter out other event types?
I think the logic below should execute only once when a the CM file is modified or re-mounted, if possible. Is it already the case?

Reply:
You're right. Previously, when we use k8s API, we only take case of event modified.
In current case, I think we only need to take care of event "REMOVE".